setup.md

```diff
@@ -39,10 +39,10 @@ to download an installer package. Please note that
 JRE or JDK installed on your system, most distribution repositories will contain OpenJRE / OpenJDK packages. 
 Install the default version available from your distribution. For example, on Ubuntu/Debian: 
 `sudo apt install default-jre`.
-* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/), 
+* OpenRefine does not support Internet Explorer. Please use [Firefox](https://www.mozilla.org/firefox/new/),
+[Microsoft Edge](https://www.microsoft.com/edge),
```

Comment:
The statement from the OpenRefine documentation says:

> OpenRefine works best on browsers based on Webkit, such as:
> * Google Chrome
> * Chromium
> * Opera
> * Microsoft Edge
> We are aware of some minor rendering and performance issues on other browsers such as Firefox. We don't support Internet Explorer.

I wonder if we might just re-use this wording as it seems clear and detailed

Reply:
I agree this Webkit language is very useful.  Should we use it in the index.md section  (as a later issue because you mentioned it's less crucial to this PR).

Reply:
Let's move this part into https://github.com/LibraryCarpentry/lc-open-refine/pull/248